crates/cext/Cargo.toml

```
@@ -0,0 +1,26 @@
+[package]
+name = "qiskit-cext"
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+
+[lib]
+name = "qiskit_cext"
+doctest = false
+crate-type = ["cdylib", "rlib"]
+
+[lints]
+workspace = true
+
+[dependencies]
+thiserror.workspace = true
+num-complex.workspace = true
+qiskit-accelerate.workspace = true
+
+[build-dependencies]
+cbindgen = "0.26"
```

Our MSRV is now 1.79 for 2.0, so we can upgrade this to the latest release (I assume that's why you held this back).
```suggestion
cbindgen = "0.28"
```

---

This was probably just generated at the time of the `0.26` release (i.e. in December which is prior to both later releases). But yes, upgrading

---

Updated 👍🏻